### PR TITLE
[NANO]: Refactor/quiescence micro spinoffs

### PR DESCRIPTION
**Pull request by ProofOfKeags**
## Change Description
Spun off from #8270 [comment](https://github.com/lightningnetwork/lnd/pull/8270#issuecomment-2260439007)

Just read the diff.

### CODE REVIEW

File: htlcswitch/test_utils.go [hunk 1]
```diff
@@ -1056,14 +1056,14 @@ func serverOptionRejectHtlc(alice, bob, carol bool) serverOption {
 // createTwoClusterChannels creates lightning channels which are needed for
 // a 2 hop network cluster to be initialized.
 func createTwoClusterChannels(t *testing.T, aliceToBob,
-	bobToCarol btcutil.Amount) (*testLightningChannel,
+	bobToAlice btcutil.Amount) (*testLightningChannel,
```
Comment (carlaKC) [inline htlcswitch/test_utils.go:1059]: While we're here, update the godoc? "2 hop network cluster" is a confusing way to refer to two sides of the same channel

**Review by carlaKC — APPROVED** [review]
🙏

**Review by yyforyongyu — APPROVED** [review]
LGTM🦾

File: htlcswitch/test_utils.go (hunk 1 above)
Comment (ProofOfKeags) [reply htlcswitch/test_utils.go:1059]: Updated to godoc *and* the name of the function.

**Pull request merged**